source/core/src/main/com/csse3200/game/components/inventory/ItemSlot.java

```diff
@@ -72,6 +74,7 @@ public ItemSlot(boolean selected) {
      */
     public void setCount(Integer count) {
         this.count = count;
+        createItemSlot();
```

Comment:
This shouldn't be too bad, I think there might be better approaches but this is nice at simple. Doesn't affect performance